test/test_utils/__init__.py

```diff
@@ -401,4 +402,13 @@ def setup_sm_benchmark_tf_train_env(resources_location, setup_tf1_env, setup_tf2
         ctx.run(f"virtualenv {venv_dir}")
         with ctx.prefix(f"source {venv_dir}/bin/activate"):
             ctx.run("pip install -U sagemaker awscli boto3 botocore six==1.11")
+
+            # SageMaker TF estimator is coded to only accept framework versions upto 2.1.0 as py2 compatible.
```

Comment:
Can you explain this change please and how is it preventing from only accepting framework versions up to 2.1.0 as py2 compatible ?

Reply:
At https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/tensorflow/estimator.py#L210
the line `_HIGHEST_PYTHON_2_VERSION = [2, 1, 0]` sets the highest TF version that can be used with python 2. This is compared against the framework version that is given as input to the `TensorFlow()` constructor in `resources/tf_sm_benchmark.py`. If the comparison shows that the framework version is higher than `2.1.0`, the API fails with an Exception.

Since TF 2.1.1 now exists, there needs to be a corresponding update to the TF API for SageMaker. However, that change will block us until that PR is merged, and deployed to pypi.

Therefore, the quick solution here was to find the location of `site-packages/` for the `sagemaker` package, and modify the `sagemaker/tensorflow/estimator.py` file in `site-packages/` to now set the `_HIGHEST_PYTHON_2_VERSION` variable to `[2, 1, 1]`.

Reply:
Thanks for the explaination.